runelite-client/src/main/java/net/runelite/client/plugins/slayer/SlayerTaskHighlightMode.java

```diff
@@ -0,0 +1,8 @@
+package net.runelite.client.plugins.slayer;
```

Comment:
missing license header. you could also just put this enum as an inner class on the config and that is also acceptable.